packages/v4/patternfly-docs/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -119,53 +150,15 @@ Once the changes initiated by the wizard are completed, a final confirmation scr
-
-
-## Usage
-
-### In-page wizard
-A wizard may be embedded in a page as shown below.
-
-<img src="./img/wizard-in-page.png" alt="In-page wizard" />
-
-
-Note that when a wizard is embedded into a page, the wizard title and description are placed within the page header. The same standards that were defined for usage of the title and description for modal wizards should be applied here.
-
-When a wizard is placed in the context of a page, it allows some freedom that is not available using the standard modal approach.
-
-* Modals may be opened from the wizard to examine required information or perform subtask that are related to, but not part of, the wizard workflow.
-* User can navigate away from the wizard to look up information or perform some prerequisite task.
-
-Be aware that anytime the user navigates aways from the wizard, there is a risk that any information entered up until that point will be lost. At a minimum, we recommend opening a modal alert when the user tries to do this warning of potential data loss and confirming that they want to leave the wizard. Optionally you should consider including a "Save as draft" feature that will allow work in progress to be saved and recalled upon returning to the wizard.
 
-### Progressive wizard
-A progressive wizard takes the same form as the standard wizard or wizard with sub-steps above but accommodates a situation where the exact number of steps is not known when the wizard is first loaded. Steps in the sidebar can be changed or added as the user progresses through the wizard.
-
-A walk-through of a progressive wizard might look like this:
-
-<img src="./img/wizard-progressive-step1.png" alt="Wizard progressive step 1" />
-
-
-Step 1: The user is presented with a Get started screen where they can specify what they want to do.
-
-<img src="./img/wizard-progressive-step2-new.png" alt="Wizard progressive step 2" />
-
-
-Step 2: Based on their choice to create a new object, a user is presented with a second set of options. But the remaining steps are still unknown.
-
-<img src="./img/wizard-progressive-step3-new.png" alt="Wizard progressive step 3" />
-
-
-Step 3: After the user chooses “Quick create” and clicks "Next," they can now be presented with a full set of steps. It should be possible to revisit either of the first two steps making different choices and steps shown should update accordingly.
-
-
-## Content
+## Content considerations
 You should keep the following in mind while designing your wizard screens:
 
 * Keep step labels short. The title at the top of each screen may be used to elaborate on the step label, but in all cases there should be a direct relationship between the step label and the screen title.
 * The last step should always be labeled review.
 * Default button labels for wizard navigation are: "Next," "Back," and "Finish." If you are replacing the default button labels, keep them short and action oriented, such as “Create network.”
+
+### Mobile considerations
```

Comment:
Don't think this should live under "Content considerations" - Could be moved to my proposed Behavior section, or at the end of the usage section